core/src/main/java/com/linecorp/armeria/server/docs/ServiceSpecification.java

```diff
@@ -212,4 +227,17 @@ public Set<ExceptionInfo> exceptions() {
     public List<HttpHeaders> exampleHeaders() {
         return exampleHeaders;
     }
+
+    /**
+     * Returns the path pattern string of the {@link DocService} mount location on server.
+     */
+    @JsonProperty
+    @Nullable
+    public Route docServiceRoute() {
```

Comment:
I don't think we will always be aware that `Route` can be serialized to a json when modifying the class.

What do you think of either:
1) Transforming `Route` to a dedicated dto
2) Adding unit tests on json serialization which ensures that the contract won't change.

Reply:
Yep, it is a good idea. Let me handle this.

Reply:
OK, I found out that in `ServiceSpecification` we serialize the following structs to JSON as well:
- HttpHeaders
- MediaType

So I think it is OK as long as I add a custom serializer to it and add a unit test rather than creating a DTO. Because we did not create DTO for any of the above.

Reply:
Hmm actually it is not really necessary to add a custom serializer either. How about just keeping `@JsonProperty` and adding unit tests?

Reply:
> How about just keeping @JsonProperty and adding unit tests?

Sure sounds good 👍